### ISSUE TITLE
Pods blocks or short codes used inside loop query are not limited to the current ID

### ISSUE THREAD

**Issue by CloundNine** [pods-framework/pods#6351]
### Description

A pods single item block placed within a loop query with Pods Name set to Use Current Pod and Slug or ID left blank (Defaults to using the current pod item) outputs "Block rendered as empty." 

The shortcode `[pods name="speaker" use_current_id="1"]{@surname}<br>[/pods] ` placed within a loop query outputs a list of all surnames (limit 15)  alphabetically. _Note I have never used 'use_current_id="1"' but successfully used 'where="id= {@ID}"' to output extra information in extended Woocommerce products, etc._

A Pods Field Value block placed within a loop query with Pods Name set to Use Current Pod and Slug or ID left blank (Defaults to using the current pod item) outputs correctly but formatting, extra characters, etc. are nearly impossible.

Pods Item List, View and Form blocks don't seem appropriate here.

### Version

2.8.5

### Testing Instructions

There aren't really any steps, but I output the speaking categories taxonomy because it was connected.  Note that Query Loop does not seem to allow the use of custom taxonomy, but applying a standard category will allow the display of only certain CPT items.  In my case I need to isolate the keynote speakers.  

### Site Health Information

```text
`
### wp-core ###

version: 5.8.2
site_language: en_US
user_language: en_US
timezone: -04:00
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 2
default_comment_status: undefined
environment_type: production
user_count: 2
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /home/icais/public_html/program
wordpress_size: 46.51 MB (48767122 bytes)
uploads_path: /home/icais/public_html/program/wp-content/uploads
uploads_size: 93.74 MB (98290687 bytes)
themes_path: /home/icais/public_html/program/wp-content/themes
themes_size: 6.69 MB (7012997 bytes)
plugins_path: /home/icais/public_html/program/wp-content/plugins
plugins_size: 134.42 MB (140951474 bytes)
database_size: 16.80 MB (17617769 bytes)
total_size: 298.16 MB (312640049 bytes)

### wp-dropins (2) ###

advanced-cache.php: true
object-cache.php: true

### wp-active-theme ###

name: Fluida Plus Child (fluida-plus-child)
version: 0.5
author: Cryout Creations
author_website: http://www.cryoutcreations.eu
parent_theme: Fluida Plus (fluida-plus)
theme_features: core-block-patterns, editor-style, widgets-block-editor, title-tag, automatic-feed-links, html5, post-formats, custom-background, custom-logo, menus, post-thumbnails, custom-header, responsive-embeds, editor-color-palette, editor-font-sizes, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, widgets
theme_path: /home/icais/public_html/program/wp-content/themes/fluida-plus-child
auto_update: Disabled

### wp-parent-theme ###

name: Fluida Plus (fluida-plus)
version: 1.8.7.1
author: Cryout Creations
author_website: http://www.cryoutcreations.eu
theme_path: /home/icais/public_html/program/wp-content/themes/fluida-plus
auto_update: Disabled

### wp-themes-inactive (1) ###

Twenty Twenty: version: 1.8, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (38) ###

Admin Columns Pro: version: 5.6.3, author: AdminColumns.com, Auto-updates enabled
Admin Columns Pro - Pods: version: 1.6.1, author: AdminColumns.com, Auto-updates enabled
Admin Columns Pro - Yoast SEO: version: 1.1, author: AdminColumns.com, Auto-updates enabled
Admin Menu Editor: version: 1.10, author: Janis Elsts, Auto-updates enabled
Advanced Editor Tools (previously TinyMCE Advanced): version: 5.6.0, author: Automattic, Auto-updates enabled
CIO Custom Fields Importer: version: 1.0.3, author: <a href="https://vipp.com.au/cio-custom-fields-importer/">VisualData</a>, Auto-updates disabled
Classic Editor: version: 1.6.2, author: WordPress Contributors, Auto-updates enabled
Content Aware Sidebars: version: 3.16.2, author: Joachim Jensen - DEV Institute, Auto-updates enabled
Enable Media Replace: version: 3.6.3, author: ShortPixel, Auto-updates enabled
Font Awesome: version: 4.0.4, author: Font Awesome, Auto-updates enabled
Force Regenerate Thumbnails: version: 2.0.6, author: Pedro Elsner, Auto-updates enabled
GDPR Cookie Consent: version: 2.0.6, author: WebToffee, Auto-updates enabled
Getwid: version: 1.7.7, author: MotoPress, Auto-updates enabled
Gravity Forms: version: 2.5.15, author: Gravity Forms, Auto-updates disabled
Imagify: version: 1.10, author: WP Media, Auto-updates enabled
LiteSpeed Cache: version: 4.4.5, author: LiteSpeed Technologies, Auto-updates disabled
Loginizer: version: 1.6.8, author: Softaculous, Auto-updates enabled
Members: version: 3.1.6, author: MemberPress, Auto-updates enabled
Plugin Notes Plus: version: 1.2.5, author: Jamie Bergen, Auto-updates enabled
Pods - Custom Content Types and Fields: version: 2.8.5, author: Pods Framework Team (latest version: 2.8.7), Auto-updates disabled
Pods Alternative Cache: version: 2.1.2, author: Pods Framework Team, Auto-updates disabled
Post Type Switcher: version: 3.2.1, author: Triple J Software, Inc., Auto-updates enabled
Redirection: version: 5.1.3, author: John Godley, Auto-updates enabled
Restricted Site Access: version: 7.2.0, author: Jake Goldman, 10up, Oomph, Auto-updates disabled
Simple History: version: 2.43.0, author: Pär Thernström, Auto-updates enabled
Simple Post Notes: version: 1.7.5, author: BracketSpace, Auto-updates enabled
Site Kit by Google: version: 1.46.0, author: Google, Auto-updates enabled
Smart Slider 3: version: 3.5.1.2, author: Nextend, Auto-updates enabled
User Switching: version: 1.5.8, author: John Blackbourn & contributors, Auto-updates disabled
User Switching in Admin Bar: version: 1.3, author: wpexpertsio, Auto-updates disabled
WP All Export Pro: version: 1.7.2, author: Soflyy, Auto-updates disabled
WP All Import Pro: version: 4.7.1, author: Soflyy, Auto-updates disabled
WP Dashboard Notes: version: 1.0.10, author: Jeroen Sormani, Auto-updates enabled
WPFront Notification Bar: version: 3.0.0.112112, author: Syam Mohan, Auto-updates enabled
WP Mail SMTP: version: 3.2.1, author: WPForms, Auto-updates enabled
WP Rollback: version: 1.7.1, author: Impress.org, Auto-updates disabled
Yoast Duplicate Post: version: 4.2, author: Enrico Battocchi & Team Yoast, Auto-updates enabled
Yoast SEO: version: 17.7.1, author: Team Yoast, Auto-updates enabled

### wp-plugins-inactive (1) ###

WP Rocket: version: 3.10.3, author: WP Media, Auto-updates enabled

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: 1808
imagemagick_version: ImageMagick 7.1.0-2 Q16 x86_64 2021-06-25 https://imagemagick.org
imagick_version: 3.5.1
file_uploads: File uploads is turned off
post_max_size: 256M
upload_max_filesize: 128M
max_effective_size: 128 MB
max_file_uploads: 20
gd_version: 2.2.5
gd_formats: GIF, JPEG, PNG, WebP, BMP, XPM
ghostscript_version: 9.25

### wp-server ###

server_architecture: Linux 3.10.0-962.3.2.lve1.5.25.12.el7.x86_64 x86_64
httpd_software: LiteSpeed
php_version: 7.4.26 64bit
php_sapi: litespeed
max_input_variables: 1000
time_limit: 60
memory_limit: 256M
max_input_time: -1
upload_max_filesize: 128M
php_post_max_size: 256M
curl_version: 7.71.0 OpenSSL/1.1.1d
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.1.48-MariaDB-cll-lve
client_version: 10.1.48-MariaDB

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/icais/public_html/program/wp-content
WP_PLUGIN_DIR: /home/icais/public_html/program/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8mb4
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### google-site-kit ###

version: 1.46.0
php_version: 7.4.26
wp_version: 5.8.2
reference_url: https://icais.org/program
amp_mode: no
site_status: connected-site
user_status: authenticated
verification_status: verified-file
connected_user_count: 1
active_modules: site-verification, search-console, analytics, analytics-4, pagespeed-insights
required_scopes: 
	openid: ✅
	https://www.googleapis.com/auth/userinfo.profile: ✅
	https://www.googleapis.com/auth/userinfo.email: ✅
	https://www.googleapis.com/auth/siteverification: ✅
	https://www.googleapis.com/auth/webmasters: ✅
	https://www.googleapis.com/auth/analytics.readonly: ✅
capabilities: 
	googlesitekit_authenticate: ✅
	googlesitekit_setup: ✅
	googlesitekit_view_posts_insights: ✅
	googlesitekit_view_dashboard: ✅
	googlesitekit_view_module_details: ✅
	googlesitekit_manage_options: ✅
enabled_features: 
	ideaHubModule: ⭕
	serviceSetupV2: ⭕
	swgModule: ⭕
	userInput: ⭕
	unifiedDashboard: ⭕
search_console_property: https://icais.org/program/
analytics_account_id: 2112•••••
analytics_property_id: UA-2112•••••••
analytics_profile_id: 2540•••••
analytics_use_snippet: yes
analytics_4_property_id: 2912416••
analytics_4_web_data_stream_id: 3017••••••
analytics_4_measurement_id: G-9Q••••••••
analytics_4_use_snippet: yes

### wp_mail_smtp ###

version: 3.2.1
license_key_type: lite
debug: No debug notices found.
db_tables: wpzm_wpmailsmtp_tasks_meta, wpzm_wpmailsmtp_debug_events
lite_install_date: Mar 10, 2021 @ 5:25pm

### pods ###

pods-server-software: LiteSpeed
pods-user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36 Edg/96.0.1054.41
pods-session-save-path: /opt/alt/php74/var/lib/php/session
pods-session-save-path-exists: Yes
pods-session-save-path-writable: Yes
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: Yes
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: Yes
pods-object-cache-redis: No
pods-memory-current-usage: 21.805M
pods-memory-current-usage-real: 24.000M
pods-network-wide: No
pods-install-location: /wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No
pods-sessions: Disable sessions
pods-can-use-sessions: Yes

`
```

### Pods Package

```json
{
    "meta": {
        "version": "2.8.5",
        "build": 1638626218
    },
    "pods": [
        {
            "name": "speaker",
            "id": 156,
            "label": "Speakers",
            "description": "",
            "label_singular": "Speaker",
            "public": "1",
            "show_ui": "1",
            "supports_title": "0",
            "supports_editor": "0",
            "type": "post_type",
            "storage": "meta",
            "publicly_queryable": "1",
            "exclude_from_search": "1",
            "capability_type": "post",
            "capability_type_custom": "speaker",
            "capability_type_extra": "1",
            "has_archive": "0",
            "hierarchical": "0",
            "rewrite": "1",
            "rewrite_with_front": "1",
            "rewrite_feeds": "0",
            "rewrite_pages": "1",
            "query_var": "1",
            "can_export": "1",
            "default_status": "draft",
            "supports_author": "0",
            "supports_thumbnail": "1",
            "supports_excerpt": "0",
            "supports_trackbacks": "0",
            "supports_custom_fields": "0",
            "supports_comments": "0",
            "supports_revisions": "0",
            "supports_page_attributes": "0",
            "supports_post_formats": "0",
            "built_in_taxonomies_category": "1",
            "built_in_taxonomies_link_category": "0",
            "built_in_taxonomies_post_tag": "0",
            "show_in_menu": "1",
            "menu_position": "0",
            "show_in_nav_menus": "1",
            "show_in_admin_bar": "1",
            "pfat_enable": "1",
            "pfat_append_single": "replace",
            "pfat_filter_single": "the_content",
            "pfat_append_archive": "append",
            "pfat_filter_archive": "the_excerpt",
            "pfat_run_outside_loop": "0",
            "rest_enable": "1",
            "read_all": "0",
            "write_all": "0",
            "pfat_single": "Single Speaker",
            "_migrated_28": "1",
            "revisions_to_keep_limit": "0",
            "delete_with_user": "1",
            "required": "0",
            "pfat_archive": "true",
            "built_in_taxonomies_speaking_category": "1",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 925,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": [
                        {
                            "name": "related_events",
                            "id": 186,
                            "label": "Related Events",
                            "description": "",
                            "weight": 0,
                            "type": "pick",
                            "pick_object": "post_type",
                            "pick_val": "event",
                            "sister_id": "154",
                            "required": "0",
                            "pick_format_type": "multi",
                            "pick_format_single": "dropdown",
                            "pick_format_multi": "list",
                            "pick_display_format_multi": "default",
                            "pick_display_format_separator": ", ",
                            "pick_allow_add_new": "0",
                            "pick_taggable": "0",
                            "pick_show_icon": "0",
                            "pick_show_edit_link": "0",
                            "pick_show_view_link": "0",
                            "pick_limit": "0",
                            "pick_user_role": "Administrator",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "1",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "post_status": "publish",
                            "pick_table": "-- Select One --"
                        },
                        {
                            "name": "surname",
                            "id": 992,
                            "label": "Surname",
                            "description": "",
                            "weight": 1,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "1",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "first_name",
                            "id": 993,
                            "label": "First Name, Initials",
                            "description": "",
                            "weight": 2,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "organization",
                            "id": 157,
                            "label": "Organization",
                            "description": "",
                            "weight": 3,
                            "type": "text",
                            "required": "0",
                            "pick_post_status": [
                                "publish"
                            ],
                            "admin_only": "0",
                            "restrict_role": "0",
                            "restrict_capability": "0",
                            "hidden": "0",
                            "read_only": "0",
                            "roles_allowed": [
                                "administrator"
                            ],
                            "rest_read": "0",
                            "rest_write": "0",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "2",
                            "unique": "0",
                            "sister_id": "-- Select One --",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255"
                        },
                        {
                            "name": "country",
                            "id": 2330,
                            "label": "Country",
                            "description": "",
                            "weight": 4,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "keynote",
                            "id": 1337,
                            "label": "Keynote",
                            "description": "",
                            "weight": 5,
                            "type": "boolean",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "boolean_format_type": "checkbox",
                            "boolean_yes_label": "Yes",
                            "boolean_no_label": "No",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1",
                            "default_value": "0"
                        },
                        {
                            "name": "job_title",
                            "id": 2528,
                            "label": "Job Title",
                            "description": "",
                            "weight": 6,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "additional_organizations",
                            "id": 2534,
                            "label": "Additional Organizations",
                            "description": "",
                            "weight": 7,
                            "type": "text",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "text_allowed_html_tags": "strong em a ul ol li b i",
                            "text_max_length": "255",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        },
                        {
                            "name": "bio",
                            "id": 2577,
                            "label": "Bio",
                            "description": "",
                            "weight": 8,
                            "type": "wysiwyg",
                            "sister_id": "-- Select One --",
                            "required": "0",
                            "wysiwyg_editor": "tinymce",
                            "roles_allowed": "administrator",
                            "rest_pick_response": "array",
                            "rest_pick_depth": "1"
                        }
                    ]
                }
            ]
        },
        {
            "name": "speaking_category",
            "id": 2678,
            "label": "Speaking Categories",
            "description": "",
            "type": "taxonomy",
            "storage": "meta",
            "label_singular": "Speaking Category",
            "public": "1",
            "show_ui": "1",
            "hierarchical": "1",
            "rest_enable": "1",
            "_migrated_28": "1",
            "built_in_post_types_speaker": "1",
            "show_admin_column": "1",
            "query_var": "1",
            "groups": [
                {
                    "name": "more_fields",
                    "id": 2679,
                    "label": "More Fields",
                    "description": "",
                    "weight": 0,
                    "fields": []
                }
            ]
        }
    ],
    "templates": [
        {
            "name": "Simple Speaker",
            "code": "{@first_name} {@surname}",
            "id": 2661,
            "_edit_lock": "1638625790:1",
            "_edit_last": "1",
            "pod_reference": {
                "pod": ""
            }
        }
    ]
}
```

**Comment by pdclark** [comment]
@CloundNine The query loop appears to work for me for embedding Speakers within a Page.

Screenshot below.

If a similar setup doesn't work for you, can you share the content of the blocks you're trying to use by clicking the three dots in the upper-right in the Gutenberg editor and selecting "Code editor"?

<img width="1404" alt="Screen Shot 2021-12-04 at 8 03 04 AM" src="https://user-images.githubusercontent.com/126889/144716266-9edf3b1c-aa9a-449f-9be7-5c692e1a3d55.png">

**Comment by CloundNine** [comment]
My code


<div class="wp-block-query">

<div class="wp-block-columns alignwide">
<div class="wp-block-column" style="flex-basis:33%"></div>



<div class="wp-block-column" style="flex-basis:66%">







</div>
</div>

</div>

**Comment by sc0ttkclark** [comment]
Fixed via 7b0cfb6d2c272d712d4e6466641317919f3600be

Previously only the Field block was working properly due to a bug in the PHP namespace usage in the other blocks. The others have been fixed now.
